### DISCUSSION TITLE
Every time you click the language switch button, the language identifier will be repeatedly added to the chrome address bar.

### DISCUSSION THREAD

**Discussion by Alan3344** [facebook/docusaurus#9295] [General]
I don’t know how to configure it to correctly switch between multi-language pages.

#### The following are the modifications made based on the original document
[Official website document](https://docusaurus.io/docs/installation)
[Local build address: http://localhost:3000/docs/intro](http://localhost:3000/docs/intro)
- `locals` and `items` are set to `zh-CN`, I also tried
![image](https://github.com/facebook/docusaurus/assets/60773912/86eb9b22-2da5-4133-83ce-9ddc72758881)

I use a script to copy the official website template to a new directory, translate it, and run `yarn start`
#### Shell script:
```bash
lang=$1
copy=$2
if [ -z "$lang" ]; then
    echo "Please enter the language name!"
    exit 1
fi

mkdir -p i18n/$lang/docusaurus-plugin-content-blog/
mkdir -p i18n/$lang/docusaurus-plugin-content-docs/
mkdir -p i18n/$lang/docusaurus-plugin-content-pages/
mkdir -p i18n/$lang/docusaurus-theme-classic/
echo {} >i18n/$lang/docusaurus-theme-classic/navbar.json
echo {} >i18n/$lang/docusaurus-theme-classic/announcementBar.json
echo {} >i18n/$lang/docusaurus-theme-classic/footer.json
echo {} >i18n/$lang/docusaurus-theme-classic/navbar.json
echo {} >i18n/$lang/code.json

if [ "$copy" = "-c" ]; then
    cp -r blog/* i18n/$lang/docusaurus-plugin-content-blog/
    cp -r docs/* i18n/$lang/docusaurus-plugin-content-docs/
    cp -r src/pages/* i18n/$lang/docusaurus-plugin-content-pages/
    cp -r src/theme/* i18n/$lang/docusaurus-theme-classic/
fi
```

#### Project structure:
![new](https://github.com/facebook/docusaurus/assets/60773912/62e76895-9152-4192-996d-857148ba436e)

![image](https://github.com/facebook/docusaurus/assets/60773912/4e5684e8-be87-4a49-884d-4efec8aae3f9)

**Comment by Josh-Cena** [comment]
See https://github.com/facebook/docusaurus/issues/8652
